examples/knative/powershell/kn-ps-vsphere-inv-slack/handler.ps1

```diff
@@ -0,0 +1,99 @@
+Function Process-Init {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   Write-Host "$(Get-Date) - Init Processing Completed`n"
+}
+
+Function Process-Shutdown {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Shutdown`n"
+
+   Write-Host "$(Get-Date) - Shutdown Processing Completed`n"
+}
+
+Function Process-Handler {
+   [CmdletBinding()]
+   param(
+      [Parameter(Position=0,Mandatory=$true)][CloudNative.CloudEvents.CloudEvent]$CloudEvent
+   )
+
+   # Decode CloudEvent
+   try {
+      $cloudEventData = $cloudEvent | Read-CloudEventJsonData -Depth 10
+   } catch {
+      throw "`nPayload must be JSON encoded"
+   }
+
+   # List of vSphere Inventory operations from a TaskEvent to process
+   $inventoryOperationsProcess = @(
```

Comment:
Can you please highlight in the README that those are the two only supported resources since you write "..., etc." in the README?